modules/build/src/main/scala/scala/build/bsp/BspImpl.scala

```diff
@@ -44,9 +62,19 @@ final class BspImpl(
     actualLocalClient.onBuildTargetDidChange(params)
   }
 
+  /** Initial setup for the Bloop project.
+    *
+    * @param currentBloopSession
+    *   the current Bloop session
+    * @param reloadableOptions
+    *   options which may be reloaded on a bsp workspace/reload request
+    * @param isBspInitRun
+    *   a flag indicating if this is the BSP initial setup workflow
+    */
```

Comment:
The name of parameter `isBspInitRun` and description should be more descriptive. 

To understand this flag, I first need to know how BSP initial setup workflow works